recipes/cartopy_offlinedata/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+$PYTHON -m pip install . -vv
+
+download_cartopy_data.sh ${PREFIX}/share/cartopy/
```

Comment:
This should probably have a `bash` at the begining of the line

Reply:
Or unless this gets installed...

Reply:
This script is part of the `cartopy_offlinedata` module, so it's in the path.

Reply:
right, sorry